minishift/minishift
# Issue #2532 Improve image export functionality

How to test
--------------

Try to export all the images from the running minishift without using this PR.

```
$ time ./minishift image export --all 
```

Now use this PR.
```
$ minishift_pr image delete --all
$ time minishift_pr image export --all
```


```
$ time ./minishift image export --all <= no longer import all the images layer
Exporting 'docker.io/openshift/origin-deployer:v3.9.0'............. OK
Exporting 'docker.io/openshift/origin-docker-registry:v3.9.0'.... OK
Exporting 'docker.io/openshift/origin-haproxy-router:v3.9.0'.... OK
Exporting 'docker.io/openshift/origin-pod:v3.9.0'.... OK
Exporting 'docker.io/openshift/origin-web-console:v3.9.0'.... OK
Exporting 'docker.io/openshift/origin:v3.9.0'... OK

real	1m8.501s
user	0m4.342s
sys	0m2.946s

$ time ./minishift image export --all  <= same here
Exporting 'docker.io/openshift/origin-deployer:v3.9.0'. OK
Exporting 'docker.io/openshift/origin-docker-registry:v3.9.0'. OK
Exporting 'docker.io/openshift/origin-haproxy-router:v3.9.0'. OK
Exporting 'docker.io/openshift/origin-pod:v3.9.0'. OK
Exporting 'docker.io/openshift/origin-web-console:v3.9.0'. OK
Exporting 'docker.io/openshift/origin:v3.9.0'. OK

real	0m13.388s
user	0m0.106s
sys	0m0.126s
```
